js/plugin/src/post-edit.js

```diff
@@ -166,6 +166,15 @@ function WPSimplechartApp( $ ) {
 	 */
 	function saveChart( data ) {
 		Object.keys( data ).forEach( function( key ) {
+			// If subtitle is set, rip it out and save it separately
+			if ('chartMetadata' === key) {
```

Comment:
Moved it up, but outside the `Object.keys( data).forEach()` - I assume we wouldn't need to check for this existing like with subtitle since it'll always be there?